app/controllers/api/v1/patients_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::PatientsController < ApplicationController
+  def sync_from_user
+    MergeRecord.bulk_merge_on_id(Patient, patient_params)
+    render json: nil, status: :ok
+  end
+
+  def sync_to_user
```

Comment:
Let's avoid using placeholder methods that do nothing.

Reply:
Just wanted to leave that specific one there so we don't forget the reasoning for the method names :) . Removed it now.
-- Srihari and Govind